test/unit/api/compress/compress_copy_test.hpp

```diff
@@ -9,6 +9,29 @@
 
 #include "test.hpp"
 
+template<typename Type> struct rvv_compress_filter
```

Comment:
Why is rvv compress filter different from any other rvv filter. This doesn't seem right

Reply:
Actually, this is the main reason why I start with compress  (and next patch will be about shuffles). They require a lot of memory to compile even with global reduction for testcases. So, for now I've created an additional filter for these tests, though it is still not clear how to make it better.